src/data/property.rs

```diff
@@ -37,6 +38,33 @@ fn host_get_property(path: &Path) -> Result<Option<Vec<u8>>, Status> {
     Ok(test::TEST_PROPERTY_VALUE.take())
 }
 
+#[cfg(test)]
+pub fn host_get_map(path: &Path) -> Result<HashMap<String, String>, String> {
+    match *path.tokens() {
+        ["request", "headers"] => Ok(HashMap::default()),
+        _ => Err(format!("Unknown map requested {:?}", path)),
+    }
+}
+
+#[cfg(not(test))]
```

Comment:
I like this way of defining the fn when it's meant to be mocked for testing and not. Might be worthy to follow the same pattern for the operation dispatcher and grpc message req/res //TODO:(didierofrivia)